src/jens/git_wrapper.py

```diff
@@ -84,3 +84,28 @@ def get_refs_exec(*args, **kwargs):
         return dict((h.name, h.commit.hexsha) for h in repo.heads)
 
     return get_refs_exec(name='show-ref', args=args, kwargs=kwargs)
+
+def rev_parse(repository_path, ref, short=None):
```

Comment:
So all functions in the module are consistent. It'd be legal to call rev_parse from another module (although right now it is only called from git_wrapper.py, IIRC), therefore it's probably a good idea to have similar interfaces.

OTOH I agree that it would be more efficient. We could make rev_parse "private" (as in prefixing it with __) and apply the change that you're suggesting.

What do you think?